src/InvariantTest.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
```

Comment:
CI is failing because this needs `forge fmt`. Also need the ABIEncoderV2 pragma  so this builds on older solc versions 
```suggestion
pragma solidity >=0.6.2 <0.9.0;
pragma experimental ABIEncoderV2;
```